purchase_order_invoicing_grouping_criteria/README.rst

```diff
@@ -0,0 +1,118 @@
+===========================================
+Purchases order invoicing grouping criteria
+===========================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Faccount--invoicing-lightgray.png?logo=github
+    :target: https://github.com/OCA/account-invoicing/tree/15.0/purchase_order_invoicing_grouping_criteria
+    :alt: OCA/account-invoicing
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/account-invoicing-15-0/account-invoicing-15-0-purchase_order_invoicing_grouping_criteria
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/95/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to use custom criteria for grouping purchases orders to be
+invoiced.
+
+Default criteria for grouping (partner, company and used currency)
+will be always applied, as if not respected, there will be business
+inconsistencies, but you can add more fields to split the invoicing according
+them.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+For creating new grouping criteria:
+
+#. Go to *Invoicing > Configuration > Management > Invoicing Grouping Criteria*.
```

Comment:
Purchase Invoicing Grouping Criteria?

Reply:
@sergio-teruel are you changing this?

Reply:
@sergio-teruel please change this for not getting confused with the sales invoicing grouping criteria.